pkg/apis/imageregistry/v1/types.go

```diff
@@ -191,38 +191,17 @@ type ImageRegistryConfigStorageS3 struct {
 	CloudFront *ImageRegistryConfigStorageS3CloudFront `json:"cloudFront,omitempty"`
 }
 
-type ImageRegistryConfigStorageAzure struct {
-	Container string `json:"container,omitempty"`
-}
-
-type ImageRegistryConfigStorageGCS struct {
-	Bucket string `json:"bucket,omitempty"`
-}
-
-type ImageRegistryConfigStorageSwift struct {
-	AuthURL   string `json:"authURL,omitempty"`
-	Container string `json:"container,omitempty"`
-	Domain    string `json:"domain,omitempty"`
-	DomainID  string `json:"domainID,omitempty"`
-	Tenant    string `json:"tenant,omitempty"`
-	TenantID  string `json:"tenantID,omitempty"`
-}
-
-type ImageRegistryConfigStorageFilesystem struct {
-	VolumeSource corev1.VolumeSource `json:"volumeSource,omitempty"`
+type ImageRegistryConfigStorageEmptyDir struct {
 }
 
 type ImageRegistryConfigStoragePVC struct {
 	Claim string `json:"claim,omitempty"`
 }
 
 type ImageRegistryConfigStorage struct {
-	Azure      *ImageRegistryConfigStorageAzure      `json:"azure,omitempty"`
-	Filesystem *ImageRegistryConfigStorageFilesystem `json:"filesystem,omitempty"`
-	GCS        *ImageRegistryConfigStorageGCS        `json:"gcs,omitempty"`
-	S3         *ImageRegistryConfigStorageS3         `json:"s3,omitempty"`
-	Swift      *ImageRegistryConfigStorageSwift      `json:"swift,omitempty"`
-	PVC        *ImageRegistryConfigStoragePVC        `json:"pvc,omitempty"`
+	EmptyDir *ImageRegistryConfigStorageEmptyDir `json:"emptydir,omitempty"`
+	S3       *ImageRegistryConfigStorageS3       `json:"s3,omitempty"`
+	PVC      *ImageRegistryConfigStoragePVC      `json:"pvc,omitempty"`
```

Comment:
Need godoc for all three (and Swift as we may keep it as Experimental)

Reply:
Would be great to have Swift marked as Experimental 👍